tests/application/namespace-label/00-application.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: skiperator.kartverket.no/v1alpha1
+kind: Application
+metadata:
+  name: test-source
+spec:
+  image: image
+  port: 8080
+  accessPolicy:
+    inbound:
+      rules:
+        - application: test-ext
+          namespacesByLabel:
+            team: skiptest
+            app: testapp-selsno
```

Comment:
`selsno`?

Reply:
**Sel**mer, **Sno**rre < My name. Just for some uniqueness

Reply:
IMO don't use your own name in the tests @SnorreSelmer, find something else to keep it unique